src/link/Elf/Atom.zig

```diff
@@ -2230,7 +2171,7 @@ const riscv = struct {
     const riscv_util = @import("../riscv.zig");
 };
 
-const ResolveArgs = struct { i64, i64, i64, i64, i64, i64, i64, i64 };
+const ResolveArgs = struct { i64, i64, i64, i64, i64, i64, i64 };
```

Comment:
why is this a tuple instead of a struct?

Reply:
It was like this before but I ought to make it a struct instead of a tuple. Will do that in a follow-up.